framework/src/base/MooseVariableScalar.C

```diff
@@ -23,6 +23,9 @@
 #include "libmesh/numeric_vector.h"
 #include "libmesh/dof_map.h"
 
+// C++
```

Comment:
I understand that grouping header files is a good thing, but I also think people already do that. That's why I think we do not need comments like "Put C++/libmesh headers here"...